spec/cache/senders/impressions_sender_spec.rb

```diff
@@ -7,6 +7,7 @@
     let(:repository) { SplitIoClient::Cache::Repositories::ImpressionsRepository.new(adapter, config) }
     let(:sender) { described_class.new(repository, config, nil) }
     let(:formatted_impressions) { sender.send(:formatted_impressions, repository.clear) }
+    let(:ip) { Socket.ip_address_list.detect { |intf| intf.ipv4_private? }.ip_address }
```

Comment:
@maksimf In split_config.rb file I saw the unknown value set and that is ok for me. My question is related to the line in impressions_sender_spec.rb ... this line:
`let(:ip) { Socket.ip_address_list.detect { |intf| intf.ipv4_private? }.ip_address }` 
what happens there if the IP could not be fetched. Is taken from config??